src/components/result-view-tab.tsx

```diff
@@ -0,0 +1,310 @@
+/**
+ * Copyright (c) 2022, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import clsx from 'clsx';
+import Tabs from '@mui/material/Tabs';
+import Tab from '@mui/material/Tab';
+import Paper from '@mui/material/Paper';
+import makeStyles from '@mui/styles/makeStyles';
+import { useIntl } from 'react-intl';
+import React, { FunctionComponent, useEffect, useMemo, useState } from 'react';
+import { SecurityAnalysisResultTab } from './security-analysis-result-tab';
+import { ShortCircuitAnalysisResultTab } from './results/shortcircuit/shortcircuit-analysis-result-tab';
+import AlertInvalidNode from './utils/alert-invalid-node';
+import {
+    UNAVAILABLE_OPTIONAL_SERVICES,
+    PARAM_DEVELOPER_MODE,
+} from '../utils/config-params';
+import { useParameterState } from './dialogs/parameters/parameters';
+import DynamicSimulationResultTab from './results/dynamicsimulation/dynamic-simulation-result-tab';
+import TabPanelLazy from './results/common/tab-panel-lazy';
+import { VoltageInitResultTab } from './voltage-init-result-tab';
+import {
+    ResultsTabsLevel,
+    ResultTabIndexRedirection,
+    useResultsTab,
+} from './results/use-results-tab';
+import { LoadFlowResultTab } from './loadflow-result-tab';
+import SensitivityAnalysisResultTab from './results/sensitivity-analysis/sensitivity-analysis-result-tab';
+import { OptionalServicesNames } from './utils/optional-services';
+import { isUnavailableService } from './utils/utils';
+
+const useStyles = makeStyles(() => ({
+    div: {
+        display: 'flex',
+    },
+    table: {
+        display: 'flex',
+        flexDirection: 'column',
+        flexGrow: 1,
+    },
+    analysisResult: {
+        display: 'flex',
+        flexDirection: 'column',
+        flexGrow: 1,
+    },
+    tabPanel: {
+        display: 'flex',
+        flexGrow: 1,
+    },
+}));
+
+interface IResultViewTabProps {
+    studyUuid: string;
+    currentNode: Partial<{ id: string }>;
+    openVoltageLevelDiagram: any;
+    resultTabIndexRedirection: ResultTabIndexRedirection;
+    disabled: boolean;
+}
+
+interface IService {
+    id: string;
+    label: string;
+    isUnavailable: boolean;
+    enableDeveloperMode: boolean;
```

Comment:
rename "enableDeveloperMode" to somethings like "shouldHide" which would be a combination of "isUnavailable" and "enableDeveloperMode" ? 

It sounds weird to have "enableDeveloperMode: true" forced for some services